src/core/lbgpu.cpp

```diff
@@ -266,6 +270,33 @@ void lb_reinit_parameters_gpu() {
       /* Eq. (81) Duenweg, Schiller, Ladd, PRE 76(3):036704 (2007). */
       lbpar_gpu.gamma_bulk[ii] = 1. - 2./(9.*lbpar_gpu.bulk_viscosity[ii]*lbpar_gpu.tau/(lbpar_gpu.agrid*lbpar_gpu.agrid) + 1.);
     }
+    
+    
+    if (lbpar_gpu.is_TRT) {
```

Comment:
Looks like you already have some of Georg's TRT changes in your code. These are not on the master yet.